README.md

```diff
@@ -92,9 +95,10 @@ the common operations for data processing and model training.
   are used as factories to instantiate the actual task-specific models from the
   configuration files dynamically.
 
-
 * Optimizers from allennlp: Implemented as children of the base `Optimizer` class.
 
+* Metric computation is supported through `jury` library. In order to fully utilize transformers' Trainer's metric computation we introduced `MetricHandler`. The default behavior of this class is to directly decode the prediction outputs and use these for metric computation which is viable for most NLP tasks; however, for some tasks in which the prediction outputs are not suitable for the given metric by default behavior, you may need to extend this class for your own task. 
```

Comment:
Either use `the` in jury library, or drop the `library` word. Also, I think we could write something likt `In order to make the metrics flexible enough to work with the trainer in a common interface` instead of `In order to fully utilize transformers' Trainer's metric computation`. Also, there should be a comma after this one regardless (before we introduced )